qiita_pet/support_files/doc/source/tutorials/account-creation.rst

```diff
@@ -1,64 +0,0 @@
-.. _account-creation:
```

Comment:
Agreed! Working on it on https://github.com/biocore/cmi-workshops side. Will ask you for review.